docker/docker-compose.yml

```diff
@@ -14,34 +14,22 @@ services:
       postgres_user: graph-node
       postgres_pass: let-me-in
       postgres_db: graph-node
-      ipfs: "ipfs:5001"
-      ethereum: "dev:http://parity:8545"
+      ipfs: 'ipfs:5001'
+      ethereum: 'dev:http://docker.host.internal:8545'
```

Comment:
Just wanted to make sure that we are aware of this issue: https://github.com/docker/for-linux/issues/264

Also check this comment: https://github.com/docker/for-linux/issues/264#issuecomment-391364563

Reply:
Ouch... thanks for pointing that out, I was hoping support for it on Linux had been added by now.

Reply:
There is a fix for this now, just tested it https://github.com/docker/for-linux/issues/264#issuecomment-965465879

I created the following PR https://github.com/graphprotocol/graph-node/pull/3031